### DISCUSSION TITLE
How to open OSystem database in console (or otherwise repair)

### DISCUSSION THREAD

**Discussion by timw** [orientechnologies/orientdb#9772] [Q&A]
I have a corrupted OSystem database obtained from a customer site, but I don't currently have the application database from that server.

I was hoping to load the OSystem database in the console to see if the repair database command could repair it, but whenever I try to connect directly to it I get the error:

> connect plocal:/repair/OSystem admin password
> Error: com.orientechnologies.orient.core.exception.OSecurityAccessException: User or password not valid for database: 'OSystem'
> 	DB name="OSystem"

Is there any way to repair a corrupted OSystem database?

**Comment by andrii0lomakin** [comment]
Hi, that is an issue introduced in the last releases we will try to fix it ASAP.

**Reply by timw** [reply]
I'm using 3.1.13, so it's not particularly cutting edge - can you point to the commit/code that caused the issue?
